logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -105,9 +105,11 @@ public void generatingMockStartGenesis(final long genesisTime, final int size) {
   }
 
   public void timeUntilGenesis(final long timeToGenesis) {
-    log.info(
-        "{} until genesis time is reached",
-        () -> DateFormatUtils.format(timeToGenesis * 1000, "hh:mm:ss"));
+    long hours = timeToGenesis / 3600;
+    long minutes = (timeToGenesis % 3600) / 60;
+    long seconds = timeToGenesis % 60;
+    String timeString = String.format("%02d:%02d:%02d", hours, minutes, seconds);
+    log.info("{} until genesis time is reached", timeString);
```

Comment:
This doesn't feel like the right solution - I suspect we just needed to specify UTC as the timezone in the call to `DateFormatUtils`:
```suggestion
    log.info("{} until genesis time is reached", () -> DateFormatUtils.formatUTC(timeToGenesis * 1000, "hh:mm:ss");
```

Reply:
Sorry I'm wrong about this.  The problem is we're trying to use a time formatter to format a duration.  We should be using `DurationFormatUtils` not `DateFormatUtils`.  Personally I quite like `formatWords` version:
```
        () -> DurationFormatUtils.formatDurationWords(timeToGenesis * 1000, true, true));
```

which gives:
`1157 days 12 hours 27 minutes 20 seconds until genesis time is reached`
or
`4 seconds until genesis time is reached`

Reply:
Done.